### DISCUSSION TITLE
How to apply Github Educational Benefits If I don't have school-issued email?

### DISCUSSION THREAD

**Discussion by domeniczz** [community/community#54157] [GitHub Education] [answered]
### Hi everyone,

**The requirement says:**

<img width="436" alt="github_education_requirements" src="https://user-images.githubusercontent.com/42294390/235462125-98fab7be-6e61-48c5-bbd5-9b63905a71c0.png">

There is an ***"or"*** between "school-issued email address" and "upload documents that prove my student status", but I don't find a way to upload the document.

---

I am study in a sino-foreign cooperatively-run college of a Chinese university, I have both Chinese and foreign university's student identity document.

But I am geographically located in China, so I am rejected when I apply with my foreign university-issued email.

And when I am trying with my Chinese university student identity, I encountered this:

<img width="434" alt="github_education_problem" src="https://user-images.githubusercontent.com/42294390/235463444-15817413-f44c-4eb2-bd8f-5afa5d1179b0.png">

What should I do now?

**Comment by jgonzalezalbisu** [comment, marked as answer]
You should enter a custom string in the school name without selecting any from the dropdown, and then you'll have the button to continue and upload evidence.
